dss/operations/flac.py

```diff
@@ -48,10 +49,12 @@ def process_keys(self):
             try:
                 flac_attributes = ddb.get_item(table=self.flac_lookup_table_name, hash_key=uuid)
             except ddb.DynamoDBItemNotFound:
+                temp_status['inDatabase'] = False
                 # nothing was found within the database
                 pass
             else:
                 temp_status.update(flac_attributes)
+                temp_status['inDatabase'] = True
```

Comment:
This is a good addition!

Reply:
yeah i thought it would be more useful than inspecting if the payload was inserted or missing. it was a pain to name tho.